src/controller/CHIPDeviceController.cpp

```diff
@@ -899,15 +902,19 @@ CHIP_ERROR DeviceCommissioner::Commission(NodeId remoteDeviceId, CommissioningPa
 
 CHIP_ERROR DeviceCommissioner::Commission(NodeId remoteDeviceId)
 {
-    // TODO(cecille): Can we get rid of mDeviceBeingCommissioned and use the remote id instead? Would require storing the
-    // commissioning stage in the device.
-    CommissioneeDeviceProxy * device = mDeviceBeingCommissioned;
+    CommissioneeDeviceProxy * device = FindCommissioneeDevice(remoteDeviceId);
     if (device == nullptr || device->GetDeviceId() != remoteDeviceId ||
```

Comment:
I guess the device ID check is no longer needed

Reply:
That is correct - thanks. Removed in next commit.